source/contributing/documentation/style-guide.md

```diff
@@ -16,12 +17,20 @@ The same holds for communication directed to contributors and maintainers:
 This is a public project, and many people will read what you write.
 Use this leverage with care.
 
-- Do: Follow the evidence-based [plain language guidelines] i.e. use simple, obvious language.
-    - Don't: Use jargon, many readers may not be familiar with technical terms.
-    - Don't: Use large words if there are smaller, simpler words that convey the same meaning.
-- Do: Use the imperative voice when giving instructions.
-    - Example: "Add the `python310` package to `buildInputs`"
-    - Don't: Use a narrative voice e.g. "Now we'll add the `python310` package to `buildInputs`".
+- Follow the evidence-based [plain language guidelines].
+
+  - Don't use jargon. Readers may not be familiar with particular technical terms.
+  - Don't use long, complicated words if there are shorter, simpler words that convey the same meaning.
+
+- Use the imperative voice when giving instructions.
+  Example:
+
+  > Add the `python310` package to `buildInputs`.
+
+  Don't use a conversational tone, as it distracts from the contents.
+  Counter-example:
```

Comment:
It's unclear what you mean by "counter-example" here. Does it mean "example refuting the previous example" or does it mean "don't do this"?